gnark-prover/main.go

```diff
@@ -290,25 +289,22 @@ func runCli() {
 					&cli.BoolFlag{Name: "json-logging", Usage: "enable JSON logging", Required: false},
 					&cli.StringFlag{Name: "prover-address", Usage: "address for the prover server", Value: "localhost:3001", Required: false},
 					&cli.StringFlag{Name: "metrics-address", Usage: "address for the metrics server", Value: "localhost:9998", Required: false},
-					&cli.StringFlag{
-						Name:     "config",
-						Aliases:  []string{"c"},
-						Usage:    "Load configuration from `FILE`",
-						Required: false,
-					},
-					&cli.BoolFlag{Name: "inclusion", Usage: "Run inclusion circuit", Required: false},
-					&cli.BoolFlag{Name: "non-inclusion", Usage: "Run non-inclusion circuit", Required: false},
-					&cli.BoolFlag{Name: "combined", Usage: "Run combined circuit", Required: false},
```

Comment:
I think for the server it makes sense to keep the combined flag so that we have the option to load the inclusion, non-inclusion and combined subsets of provingkeys separately

Reply:
i think this is an implementation detail that we can address in a follow-up pr. wdyt?